ValidationError: 2 validation errors for PaymentResponse
created_at
  Field required [type=missing, input_value={'amount': 5999, 'order_u...46c6-807f-f62e6f0c54da'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
updated_at
  Field required [type=missing, input_value={'amount': 5999, 'order_u...46c6-807f-f62e6f0c54da'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
